content/en/docs/kitex/Tutorials/basic-feature/transport_protocol.md

```diff
@@ -5,4 +5,65 @@ weight: 3
 description: >
 ---
 
-Translation TODO
+Usually the RPC protocol contains RPC message protocol and application layer transport protocol, the RPC message protocol is seen as the Payload of the transport message, the transport protocol passes some additional meta information usually used for service governance, the framework of MetaHandler is also used with the transport protocol. In the microservice scenario, transport protocols play an important role, such as link tracking of transmissions information is usually passed by the transport protocol for the link.
+
+Kitex currently supports two transport protocols: TTHeader, HTTP2, but the actual Transport Protocols provided for configuration are: TTHeader, GRPC, Framed, TTHeaderFramed, PurePayload. here are some notes: because Kitex support for Protobuf support Kitex Protobuf and gRPC, to facilitate the distinction between gRPC as the classification of transport protocols, the framework will decide which protocol to use according to whether there is a configuration of gRPC; Framed is not strictly speaking a transport protocol, but only an additional 4-byte header to mark the Payload Size, but the message protocol is not mandatory for whether there is a Framed header or not. Framed is not strictly speaking a transport protocol, but the message protocol is not mandatory for Framed headers, PurePayload does not have any header, so Framed is also classified as a transport protocol.
```

Comment:
1. provided for configuration -> provided with configuration
2. here are some notes -> Here ...
3. because Kitex support for Protobuf support Kitex Protobuf and gRPC  -> totolly wrong translation

Reply:
Here... 不是说就需要有个 Here 就可以了，是告诉你句子开头需要大写，其它的不是说要删掉。